### ISSUE TITLE
Forthcoming spdep addresses much of the content of this package

### ISSUE THREAD

**Issue by rsbivand** [horankev/sfislands#2]
1. The reference in your arxiv paper to `spdep` is not accurate.
2. You should reference: Bivand R, Portnov B, (2004) Exploring spatial data analysis techniques using R: the case of observations with no neighbours. In: Anselin L, Florax R, Rey S, (eds.), \emph{Advances in Spatial Econometrics, Methodology, Tools and Applications}. Berlin: Springer-Verlag, pp. 121-142.
3. Subgraphs are a much larger problem than islands, forthcoming `spdep` makes these much more visible. They were already checked by e.g. `CARBayes` in the ICAR setting. See also https://doi.org/10.1016/j.sste.2018.04.002.

**Comment by horankev** [comment]
Thanks for your comments to help to improve the paper. I have implemented all of your suggestions. The reference to the `spdep` package has been corrected. Also, I have directed the reader to the other papers you mentioned for a deeper discussion of dealing with disconnected units/subgraphs. I have emphasised that this package offers an convenient way to visualise, add and remove links for situations where that is deemed an appropriate approach.
